src/snowflake/snowpark/session.py

```diff
@@ -191,6 +191,16 @@ class Session:
     A :class:`Session` object is not thread-safe.
     """
 
+    class RuntimeConfig:
```

Comment:
I might miss something but it does not seem like we have a public method to add entries? Should `_conf` be populated in `__init__`?